src/app/containers/MostRead/index.jsx

```diff
@@ -7,8 +7,8 @@ import Canonical from './Canonical';
 
 const getMostReadEndpoint = ({ service, variant }) =>
   variant
-    ? `/${service}/most_read/${variant}.json`
-    : `/${service}/most_read.json`;
+    ? `/${service}/mostread/${variant}.json`
+    : `/${service}/mostread.json`;
 
 const MostReadContainer = ({ endpointOverride }) => {
```

Comment:
Is there any logic to only render the Canonical component on canonical pages?

Reply:
not atm 👀

Reply:
just added in some logic now, to enable most read when it's not AMP 🤞